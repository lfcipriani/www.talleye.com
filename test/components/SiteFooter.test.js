import renderer from 'react-test-renderer';
import SiteFooter from '../../components/SiteFooter';

beforeEach(() => {
  const useRouter = jest.spyOn(require('next/router'), 'useRouter');
  useRouter.mockImplementation(() => ({
    locale: 'en',
    defaultLocale: 'en',
  }));
});

describe('SiteFooter snapshot', () => {
  it('renders Footer', () => {
    const component = renderer.create(<SiteFooter />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
