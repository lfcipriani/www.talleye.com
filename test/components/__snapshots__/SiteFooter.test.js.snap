// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteFooter snapshot renders Footer 1`] = `
<footer
  className="jsx-1480459494"
>
  <div
    className="jsx-1480459494"
  >
    <p
      className="jsx-1480459494"
    >
      <a
        className="jsx-1480459494"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Tall Eye
      </a>
      , 
      a website by
       
      <a
        className="jsx-1480459494"
        href="/about"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Luis Cipriani
      </a>
      <br
        className="jsx-1480459494"
      />
      <a
        href="https://creativecommons.org/licenses/by-nc/4.0/"
        rel="noreferrer nofollow"
        target="_blank"
      >
        Some terms apply
      </a>
    </p>
  </div>
  <div
    className="jsx-1480459494"
  >
    <ul
      className="jsx-1480459494"
    >
      <li
        className="jsx-1480459494"
      >
        <a
          className="jsx-3248159136"
          href="/posts-rss.xml"
          onClick={[Function]}
          onMouseEnter={[Function]}
          target="_blank"
          title="RSS feed"
        >
          <span
            className="jsx-3248159136 visually-hidden"
          >
            RSS feed
          </span>
          <svg
            aria-hidden="true"
            className="jsx-3248159136 svgIcon"
            focusable="false"
            height="35"
            viewBox="0 0 256 256"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              className="jsx-3248159136"
              fill="none"
              height="256"
              width="256"
            />
            <path
              className="jsx-3248159136 stroke"
              d="M48,144a64,64,0,0,1,64,64"
              fill="none"
              stroke="#284B63"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <path
              className="jsx-3248159136 stroke"
              d="M48,96A112,112,0,0,1,160,208"
              fill="none"
              stroke="#284B63"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <path
              className="jsx-3248159136 stroke"
              d="M48,48A160,160,0,0,1,208,208"
              fill="none"
              stroke="#284B63"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <circle
              className="jsx-3248159136 fill"
              cx="52"
              cy="204"
              r="12"
            />
          </svg>
        </a>
      </li>
      <li
        className="jsx-1480459494"
      >
        <a
          href="#"
          onClick={[Function]}
          title="Toggle to dark theme"
        >
          <span
            className="jsx-676870541 visually-hidden"
          >
            Toggle to dark theme
          </span>
          <svg
            aria-hidden="true"
            className="jsx-676870541 svgIcon"
            fill="#284B63"
            focusable="false"
            height="35"
            viewBox="0 0 256 256"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              className="jsx-676870541"
              fill="none"
              height="256"
              width="256"
            />
            <path
              className="jsx-676870541 stroke"
              d="M88,148a68,68,0,1,1,68,68H76a44,44,0,1,1,14.30583-85.62208"
              fill="none"
              stroke="#284B63"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <path
              className="jsx-676870541 stroke"
              d="M46.99,138.92889A64.12919,64.12919,0,0,1,9.6218,94.37955l.00023-.001A64.0566,64.0566,0,0,0,86.37854,17.622l.00084-.00019A64.02478,64.02478,0,0,1,136,80q0,1.51431-.06961,3.01174"
              fill="none"
              stroke="#284B63"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
          </svg>
        </a>
      </li>
      <li
        className="jsx-1480459494"
      >
        <a
          className="jsx-3248159136"
          href="#top"
          title="Scroll to the top of the page"
        >
          <span
            className="jsx-3248159136 visually-hidden"
          >
            Scroll to the top of the page
          </span>
          <svg
            aria-hidden="true"
            className="jsx-3248159136 svgIcon"
            fill="#0e1007"
            focusable="false"
            height="35"
            viewBox="0 0 256 256"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              className="jsx-3248159136"
              fill="none"
              height="256"
              width="256"
            />
            <circle
              className="jsx-3248159136 stroke"
              cx="128"
              cy="128"
              fill="none"
              r="96"
              stroke="#0e1007"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <polyline
              className="jsx-3248159136 stroke"
              fill="none"
              points="96 168 128 136 160 168"
              stroke="#0e1007"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
            <polyline
              className="jsx-3248159136 stroke"
              fill="none"
              points="96 112 128 80 160 112"
              stroke="#0e1007"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="16"
            />
          </svg>
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
